### ISSUE TITLE
  main函数运行到这一步的时候报错 self.neighbors.fit(embedding_matrix)

### ISSUE THREAD

**Issue by yuexinshen** [MingzeLucasNi/FBA#1]
Can't call numpy() on Tensor that requires grad. Use tensor.detach().numpy() instead.

**Comment by MingzeLucasNi** [comment]
Hi, thanks for reporting. You might need to run python bert_emd_create.py first to generate the word embedding.
